editor/src/templates/editor-canvas.tsx

```diff
@@ -1104,12 +1036,16 @@ export class EditorCanvas extends React.Component<EditorCanvasProps> {
       <EditorCommon
         mouseDown={this.handleMouseDown}
         mouseUp={this.handleMouseUp}
-        keyDown={this.handleKeyDown}
-        keyUp={this.handleKeyUp}
+        keyDown={this.handleKeyNoOp}
+        keyUp={this.handleKeyNoOp}
```

Comment:
Do we need these?

Reply:
good catch, this was meant to be temporary

Reply:
fixed